context/app/static/js/components/Header/Menu/Menu.jsx

```diff
@@ -29,6 +29,7 @@ function Menu(props) {
             {['Donor', 'Sample', 'Dataset'].map((type) => (
```

Comment:
```suggestion
            {/* NOTE: Changes here should be in sync with HeaderContent.jsx. */}
            {['Donor', 'Sample', 'Dataset'].map((type) => (
```

... and similarly in the other file. I had started to do this in my branch, but forgot that there were two places to change.